website/content/docs/configuration/server.mdx

```diff
@@ -267,6 +267,14 @@ server {
 - `job_tracked_versions` `(int: 6)` - Specifies the number of historic job versions that
   are kept.
 
+- `oidc_issuer` `(string: "")` - Specifies the Issuer URL for [Workload
+  Identity][wi] JWTs. If set the `/.well-known/openid-configuration` HTTP
+  endpoint is enabled for third parties to discover Nomad's OIDC configuration.
+  Once set `oidc_issuer` *cannot be changed* without invalidating Workload
```

Comment:
How bad would it be for this config to drift across servers, like a typo or an old machine coming back up for some reason?

Reply:
Yeah, I was wondering about storing this in state. I don't have any strong opinions, and I don't know if we have any prescriptive guidance of what should go where 🤔 

Some aspects I just pulled out of thin air:

| Config file | State |
| ---------- | ----- |
| Required on agent start | Not required on agent start |
| Specific to each node | Must be the same in all nodes |
| Requires restart to apply change | Can be changed live |

So I feel like this falls into the "state" category? But I also don't feel strongly about my table either 😅

Reply:
Yeah, a more holistic strategy for configuration management would be great!

I'm good with the changes here as they are, I just smelled a potentially bad footgun and wanted to confirm it was indeed a bad problem 😅